src/MizarWidgets/include/MizarWidgets/SamplingWithFrameTrackReportConfigValidator.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_WIDGETS_SAMPLING_WITH_FRAME_TRACK_REPORT_CONFIG_VALIDATOR_
+#define MIZAR_WIDGETS_SAMPLING_WITH_FRAME_TRACK_REPORT_CONFIG_VALIDATOR_
+
+#include <absl/functional/bind_front.h>
+
+#include <QMessageBox>
+#include <QString>
+
+#include "MizarBase/BaselineOrComparison.h"
+#include "MizarData/BaselineAndComparison.h"
+#include "MizarData/MizarPairedData.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+#include "OrbitBase/Result.h"
+
+namespace orbit_mizar_widgets {
+
+template <typename BaselineAndComparison, typename PairedData, auto ReportError>
```

Comment:
Please add a high-level description of the class here.

Reply:
What does malformed means? I guess you could add a sentence about this here. Also you should mention that you should call `Validate`.